projects/bp-gallery/src/components/views/start/DailyPicture.tsx

```diff
@@ -73,22 +90,14 @@ const DailyPicture = () => {
   return (
     <div>
       {picture && (
-        <>
-          <h3 className={'text-2xl'}>{t('common.daily-picture')}</h3>
-          <div className={'flex flex-col place-items-center'}>
-            <img
-              className={'w-screen sm:w-auto sm:h-50vh'}
-              src={pictureLink}
-              alt={t('common.daily-picture')}
-              onClick={async () => {
-                await navigateToPicture(picture.id);
-              }}
-            />
-            <div className={'flex flex-col max-w-4xl'}>
-              <p className={'line-clamp-5'}>
-                <h4 className={'text-lg my-1'}>{t('pictureFields.descriptions')}:</h4>
+        <div className='flex justify-center'>
+          <Card className='flex flex-col-reverse md:flex-row rounded-md justify-between max-w-4xl max-h-fit'>
+            <div className='p-4 flex flex-col'>
+              <h3 className={'text-2xl'}>{t('common.daily-picture')}</h3>
+              <div className={'line-clamp-5'}>
+                <h4 className={'text-lg my-1'}>{t('common.description')}:</h4>
                 <RichText value={description} />
```

Comment:
```suggestion
                <RichText value={description} className='break-all' />
```
Else too long single lines will break the layout, for example: 
![image](https://user-images.githubusercontent.com/64468276/221202469-e22095f6-8644-4fc3-a708-7c2c13343c39.png)